src/Context/GetoptParser.php

```diff
@@ -265,12 +270,14 @@ protected function splitLongOptionInput($input)
      * @return bool
      *
      */
-    protected function longOptionRequiresValue($option, $value, $name)
+    protected function longOptionRequiresValue($option, $value, $name=null)
     {
         if ($option->param == 'required' && trim($value) === '') {
-            $this->errors[] = new Exception\OptionParamRequired(
-                "The option '$name' requires a parameter."
-            );
+            if ($name) {
```

Comment:
I think @pmjones style is also similar. I am looking at https://github.com/auraphp/Aura.Cli/blob/2.x/src/Help.php#L298 .